src/IIS/Settings/Bindings/FluentAPI/IBindings.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Cake.IIS.Settings.Bindings.FluentAPI
+{
+    public interface IBindings
```

Comment:
Why use an interface when your effectively trying to mimic an Enum of classes? Drop the interface and just use a static class to expose the individual implementations